chroma_feedback/consumer/razer_chroma/device.py

```diff
@@ -17,6 +17,14 @@ def process_devices(devices : Any, status : str) -> List[Dict[str, Any]]:
 	# process devices
 
 	for device in devices:
+		# Determine the lighting effect based on device capabilities:
+		if device.fx.has('logo_pulsate') and device.fx.has('scroll_pulsate'):
+			effect_method = "pulsate_device"
+		elif device.fx.has('logo_breath_single') and device.fx.has('scroll_breath_single'):
+			effect_method = "breath_device"
+		else:
+			effect_method = "pulsate_device"
```

Comment:
This fallback is here to catch devices which may not fall in either broad category (pulsation support or breath support), or somehow don't support both a logo and a scroll wheel.

In the future, it might be useful to configure effects based on the full set of capabilities exposed by openrazer; if that happens, this block has to expand and probably undergo a rename.